assets/scss/modules/_forms.scss

```diff
@@ -0,0 +1,391 @@
+    max-width: 306px;
+    overflow-x: hidden;
+
+    .country {
+      display: flex;
+      align-items: center;
+
+      &:after {
+        content: '';
+        position: absolute;
+        height: 35px;
+        width: 1px;
+        left: 33%;
+        border-left: 1px solid $dp-color-grey;
+      }
+
+      .flag-box {
+        order: 1;
+      }
+
+      .dial-code {
+        order: 2;
+        align-items: center;
+        min-width: 80px;
+        font-size: $dp-font-size-p;
+        font-style: italic;
+      }
+
+      .country-name {
+        order: 3;
+        font-size: $dp-font-size-p;
+        color: $dp-color-grey;
+        font-style: italic;
+      }
+    }
+  }
+
+  .holder {
+    position: relative;
+    display: inline-block;
+    text-align: left;
+
+    &.big {
+      width: 100%;
+      padding-right: 0;
+      margin-bottom: 25px;
+
+      input,
+      textarea {
+        width: 100%;
+      }
+    }
+
+    label {
+      color: $dp-color-grey;
+      font-size: $dp-font-size-p;
+      font-weight: bold;
+      line-height: 15px;
+      padding-bottom: 0;
+      display: inline-block;
+
+      & span {
+        font-weight: normal;
+        font-style: italic;
+        color:$dp-color-lightgrey;
+        font-size: $dp-font-size-small;
+      }
+    }
+
+    &.checkbox {
+      cursor: pointer;
+      display: flex;
+      align-items: center;
+
+      label {
+        margin-left: -40px;
+        padding-bottom: 0;
+      }
+
+      input[type='checkbox'] {
+        height: 40px;
+        opacity: 0;
+        z-index: 0;
+        cursor: pointer;
+        margin-left: -20px;
+        flex: 0 1 80px;
+
+        &:checked {
+          & ~ .checkmark {
+            border-color: $dp-color-green;
+            background-color: $dp-color-green;
+
+            &:after {
+              display: block;
+            }
+          }
+        }
+      }
+
+      .checkmark {
+        position: absolute;
+        left: 0;
+        height: 16px;
+        width: 16px;
+        border: 1px solid $dp-color-grey;
+        border-radius: 3px;
+        background-color: $dp-color-white;
+        pointer-events: none;
+
+        &:after {
+          content: '';
+          position: absolute;
+          left: 5px;
+          top: 1px;
+          width: 5px;
+          height: 10px;
+          border: solid $dp-color-white;
+          border-width: 0 3px 3px 0;
+          transform: rotate(45deg);
+        }
+      }
+
+      label {
+        font-weight: normal;
+      }
+
+      &.error {
+        .checkmark {
+          border-color: $dp-color-red;
+        }
+      }
+    }
+  }
+
+  input:not(.dp-button):not([type='checkbox']),
+  textarea {
+    height: 40px;
+    min-width: 370px;
+    border: 1px solid $dp-color-grey;
+    border-radius: 3px;
+    background-color: $dp-color-white;
+    font-size: $dp-font-size-p;
+    line-height: 20px;
+    padding: 10px 12px;
+    margin-top: 5px;
+
+    &::placeholder {
+      color: #838370;
```

Comment:
Could you add this `#838370` into a sass var?

Reply:
use variables :)